README.md

```diff
@@ -1,18 +1,43 @@
-ublas
+Boost.uBLAS Linear Algebra Library
 =====
+Boost.uBLAS is part of the [Boost C++ Libraries](http://github.com/boostorg). It is directed towards scientific computing on the level of basic linear algebra constructions with matrices and vectors and their corresponding abstract operations. 
 
-# Building [![Build Status](https://travis-ci.org/boostorg/ublas.svg?branch=develop)](https://travis-ci.org/boostorg/ublas) [![Build status](https://ci.appveyor.com/api/projects/status/ctu3wnfowa627ful/branch/develop?svg=true)](https://ci.appveyor.com/project/stefanseefeld/ublas/branch/develop)
+## Fork
```

Comment:
The docs need to be updated to reflect that this is no longer a fork. :-)

Reply:
Yes. Will change that.

Reply:
Done.